src/edu/csus/ecs/pc2/ui/TestResultsPane.java

```diff
@@ -1003,6 +1012,13 @@ private JPanel getResultsPaneButtonPanel() {
             // add a control button to invoke comparison of the team and judge output files for selected row(s)
             resultsPaneButtonPanel.add(getCompareSelectedButton());
 
+            //add space
+            Component horizontalStrut_4 = Box.createHorizontalStrut(20);
+            resultsPaneButtonPanel.add(horizontalStrut_4);
+            
+            // add a control button to invoke comparison of the team and judge output files for selected row(s)
+            resultsPaneButtonPanel.add(getExecuteAllButton());
```

Comment:
It might be a good idea to only add this button if the problem has "stop at first error" set.  If "stop at first error" is not set, then this button is meaningless (and confusing).

Reply:
Updated